doc/Status.md

```diff
@@ -1,58 +0,0 @@
-The Sail specification currently captures the following ISA features:
-
-- The RV32I and RV64I primary base ISAs.
-
-- The M (multiply/divide), A (atomic), and C (compressed) Standard
-  Extensions.
-
-- The F (single-precision) and D (double-precision) Floating-Point
-  Standard Extensions.  This is only executable in the C emulator,
-  which uses the SoftFloat C library.
-
-- The Zicsr Control and Status Register Standard Extension.
-
-- The N Standard Extension for User-Level Interrupts.
-
-- The Base Counters and Timers.
-
-- The Machine-Level and Supervisor-Level ISAs for RV32 and RV64.
-
-- Physical Memory Protection (PMP)
-
-For the RVWMO memory consistency model, this Sail ISA semantics is integrated with the RVWMO operational model in [the
-RMEM tool](https://github.com/rems-project/rmem).
-
-The Sail specification is parameterized over the following
-platform-specific options:
-
-- handling of misaligned data accesses with or without M-mode traps.
-
-- updating of the PTE dirty bit with or without architectural
-  exceptions.
-
-- the contents of the `mtval` register on an illegal instruction
-  exception.
-
-The following ISA features are specified in the
```

Comment:
I think it would be worth keeping this list of notable omissions: 

* Hypervisor
* Mutable MXLEN/SXLEN
* Big endian
* RV32E
* PMAs

I don't think we need to mention RV128 or RV64E since I'm pretty sure they don't exist. Or hardware counters because that's kind of expected and fairly minor.

It would be cool to get a table of the RVA22/23 profiles with ticks and crosses for our support, but we don't need to in this PR.

Reply:
Added a section with these omissions. RV64E is ratified, so I included it with RV32E. There are a lot of other extensions/features that are not supported at the moment so it's hard to say where to draw the line, but for now we can just go with this list.